dsgrid/config/dimensions.py

```diff
@@ -546,6 +546,14 @@ class DateTimeDimensionModel(TimeDimensionBaseModel):
             },
         ),
     ]
+    wrap_time_allowed: Annotated[
```

Comment:
I agree. Probably the project coordinator should indicate whether they want to wrap time for particular datasets.

Reply:
But looking at more of the code below, I don't think the project coordinator should have to flip any switches to wrap time just for period-beginning/period-ending. Maybe there should be a project config that specifies a maximum amount of time-wrapping (in number of timestamps and/or hours)?

Reply:
Good suggestion to move this parameter to InputDatasetModel. I think it makes sense to allow time wrap automatically if it's only needed to align time interval type and have "wrap_time_allowed" to handle the particular case where the time zones are misaligned? It might be too confusing for the coordinator to limit the max amount of time-wrapping, at least I can't think of a concrete use case for that.

Reply:
That sounds good to me.